packages/expo-cli/src/commands/eas-build/utils/expoUpdates/android.ts

```diff
@@ -0,0 +1,232 @@
+      "Could not find the build script for Android. This could happen in case of outdated 'node_modules'. Run 'npm install' to make sure that it's up-to-date."
+    );
+  }
+
+  return `apply from: ${JSON.stringify(
+    path.relative(path.join(projectDir, 'android', 'app'), androidBuildScriptPath)
+  )}`;
+}
+
+export async function setUpdatesVersionsAndroidAsync({
+  projectDir,
+  exp,
+}: {
+  projectDir: string;
+  exp: ExpoConfig;
+}) {
+  if (!isExpoUpdatesInstalled(projectDir)) {
+    return;
+  }
+
+  const isUpdatesConfigured = await isUpdatesConfiguredIfInstalledAndroidAsync(projectDir);
+
+  if (!isUpdatesConfigured) {
+    throw new CommandError(
+      '"expo-updates" is installed, but not configured in the project. Please run "expo eas:build:init" first to configure "expo-updates"'
+    );
+  }
+
+  const {
+    path: androidManifestPath,
+    data: androidManifestJSON,
+  } = await getAndroidManifestJSONAsync(projectDir);
+
+  const runtimeVersion = AndroidConfig.Updates.getRuntimeVersion(exp);
+  const sdkVersion = AndroidConfig.Updates.getSDKVersion(exp);
+
+  const currentRuntimeVersion = getAndroidMetadataValue(
+    androidManifestJSON,
+    AndroidConfig.Updates.Config.RUNTIME_VERSION
+  );
+
+  const currentSdkVersion = getAndroidMetadataValue(
+    androidManifestJSON,
+    AndroidConfig.Updates.Config.SDK_VERSION
+  );
+
+  if (
+    (runtimeVersion && runtimeVersion === currentRuntimeVersion) ||
+    (sdkVersion && sdkVersion === currentSdkVersion)
+  ) {
+    return;
+  }
+
+  const result = await AndroidConfig.Updates.setVersionsConfig(exp, androidManifestJSON);
+
+  await AndroidConfig.Manifest.writeAndroidManifestAsync(androidManifestPath, result);
+}
+
+export async function configureUpdatesAndroidAsync({
+  projectDir,
+  exp,
+}: {
+  projectDir: string;
+  exp: ExpoConfig;
+}) {
+  if (!isExpoUpdatesInstalled(projectDir)) {
+    return;
+  }
+
+  const username = await UserManager.getCurrentUsernameAsync();
+  const buildGradlePath = getAndroidBuildGradlePath(projectDir);
+  const buildGradleContent = await getAndroidBuildGradleContentAsync(buildGradlePath);
+
+  if (!hasBuildScriptApply(buildGradleContent, projectDir, exp)) {
+    const androidBuildScript = getAndroidBuildScript(projectDir, exp);
+
+    await fs.writeFile(
+      buildGradlePath,
+      `${buildGradleContent}\n// Integration with Expo updates\n${androidBuildScript}\n`
+    );
+  }
+
+  const {
+    path: androidManifestPath,
+    data: androidManifestJSON,
+  } = await getAndroidManifestJSONAsync(projectDir);
+
+  if (!isMetadataSetAndroid(androidManifestJSON, exp, username)) {
+    const result = await AndroidConfig.Updates.setUpdatesConfig(exp, androidManifestJSON, username);
+
+    await AndroidConfig.Manifest.writeAndroidManifestAsync(androidManifestPath, result);
+  }
+}
+
+async function isUpdatesConfiguredIfInstalledAndroidAsync(projectDir: string) {
+  if (!isExpoUpdatesInstalled(projectDir)) {
+    return true;
+  }
+
```

Comment:
```suggestion
async function isUpdatesConfiguredAsync(projectDir: string) {
```
If you only use that function in `setUpdatesVersionsAndroidAsync`, then you can skip this check and revert back to old name

Reply:
This is not the change I suggested